onnxruntime/core/framework/session_state.h

```diff
@@ -141,17 +141,18 @@ class SessionState {
      * \param p_node0 Nullable
      * \param kci0 Nullable
      */
-    NodeInfo(size_t index0, const onnxruntime::Node* p_node0, const KernelCreateInfo* kci0)
-        : index(index0),
-          p_node(p_node0),
-          kci(kci0) {
-    }
+    NodeInfo(size_t index_, const onnxruntime::Node* p_node_, const KernelCreateInfo* kci_, const OrtDevice& device_)
```

Comment:
> index_ [](start = 19, length = 7)

Is the convention to have members with trailing underscore? This is confusing. p_ prefix is sufficient if used consistently.

Reply:
I thought this was from google coding standard. no?